docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -0,0 +1,263 @@
+
+{{< docs/shared lookup="flow/reference/components/otelcol-compression-field.md" source="agent" >}}
+
+The `balancer_name` argument controls what client-side load balancing mechanism
+to use. See the gRPC documentation on [Load balancing][] for more information.
+When unspecified, `pick_first` is used.
+
+An HTTP proxy can be configured through the following environment variables:
+
+* `HTTPS_PROXY`
+* `NO_PROXY`
+
+The `HTTPS_PROXY` environment variable specifies a URL to use for proxying
+requests. Connections to the proxy are established via [the `HTTP CONNECT`
+method][HTTP CONNECT].
+
+The `NO_PROXY` environment variable is an optional list of comma-separated
+hostnames for which the HTTPS proxy should _not_ be used. Each hostname can be
+provided as an IP address (`1.2.3.4`), an IP address in CIDR notation
+(`1.2.3.4/8`), a domain name (`example.com`), or `*`. A domain name matches
+that domain and all subdomains. A domain name with a leading "."
+(`.example.com`) matches subdomains only. `NO_PROXY` is only read when
+`HTTPS_PROXY` is set.
+
+Because `otelcol.exporter.loadbalancing` uses gRPC, the configured proxy server must be
+able to handle and proxy HTTP/2 traffic.
+
+[Load balancing]: https://github.com/grpc/grpc-go/blob/master/examples/features/load_balancing/README.md#pick_first
+[HTTP CONNECT]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/CONNECT
+
+### tls block
+
+The `tls` block configures TLS settings used for the connection to the gRPC
+server.
+
+{{< docs/shared lookup="flow/reference/components/otelcol-tls-config-block.md" source="agent" >}}
+
+### keepalive block
+
+The `keepalive` block configures keepalive settings for gRPC client
+connections.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`ping_wait` | `duration` | How often to ping the server after no activity. | | no
+`ping_response_timeout` | `duration` | Time to wait before closing inactive connections if the server does not respond to a ping. | | no
+`ping_without_stream` | `boolean` | Send pings even if there is no active stream request. | | no
+
+### queue block
+
+The `queue` block configures an in-memory buffer of batches before data is sent
+to the gRPC server.
+
+{{< docs/shared lookup="flow/reference/components/otelcol-queue-block.md" source="agent" >}}
+
+### retry block
+
+The `retry` block configures how failed requests to the gRPC server are
+retried.
+
+{{< docs/shared lookup="flow/reference/components/otelcol-retry-block.md" source="agent" >}}
+
+## Exported fields
+
+The following fields are exported and can be referenced by other components:
+
+Name | Type | Description
+---- | ---- | -----------
+`input` | `otelcol.Consumer` | A value that other components can use to send telemetry data to.
+
+`input` accepts `otelcol.Consumer` data for telemetry signals of these types:
```

Comment:
This way of presenting the exported fields is not standard. I customised it to highlight that only logs and traces work. Maybe I also mention unsupported types like this:

> These signal types are not supported:
> * metrics

We also have #4151 for this presentation issue